### ISSUE TITLE
Adding v5 Tag to repository

### ISSUE THREAD

**Issue by HenryAsa** [github/super-linter#4083]
## Describe the bug

With the release of `v5.0.0`, a "`v5`" tag should be added, but it does not yet exist.

Kind of related to #4082

## Expected behavior

The `v5.0.0` tag exists, but the `v5` tag does not yet exist.  As a result, I cannot automatically reference the latest `v5` version of the code.

## Steps to Reproduce

1. In any `.github/workflows/linter.yml`, running the following code returns an error:

```
jobs:
  build:
    name: Lint Code Base
    runs-on ubuntu-latest
    steps:
      ###########################
      # Checkout the code base #
      ##########################
      - name: Checkout Code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0

      ################################
      # Run Linter against code base #
      ################################
      - name: Lint Code Base
         uses: github/super-linter@v5
```

This returns:

> Unable to resolve action `github/super-linter@v5`, repository or version not found

**Comment by bowenliang123** [comment]
Found the same blocking problem when using `v5` in Kyuubi's CI with https://github.com/apache/kyuubi/pull/4695. 

https://github.com/apache/kyuubi/actions/runs/4687897085/jobs/8307763429?pr=4695#step:1:35
```
Error: Unable to resolve action `github/super-linter@v5`, unable to find version `v5`
```

**Comment by lindluni** [comment]
This has been fixed, apologies on this, we don't do major version often and this was the first one with me leading the project, we get the automation fixed for the future to prevent these situations from happening.

**Comment by bowenliang123** [comment]
Thanks for the fix. Good to have a quick response from you and your team. And even without `v5` tag, it's also available with `v4` tag in `v4.10.2` on this release.
